youtube_dl/extractor/dailymotion.py

```diff
@@ -106,9 +107,9 @@ def _real_extract(self, url):
         age_limit = self._rta_search(webpage)
 
         video_upload_date = None
-        mobj = re.search(r'<div class="[^"]*uploaded_cont[^"]*" title="[^"]*">([0-9]{2})-([0-9]{2})-([0-9]{4})</div>', webpage)
+        mobj = re.search(r'<meta property="video:release_date" content="([0-9]{4})-([0-9]{2})-([0-9]{2}).+?"/>',webpage)
```

Comment:
It's common in python code to add a space after ',' (see [the PEP 8](https://www.python.org/dev/peps/pep-0008/) for the style guide we try to follow).